### ISSUE TITLE
luasec stub has a hardcoded ca-file path

### ISSUE THREAD

**Issue by ikisusi** [ouspg/trytls#118]
There is a hardcoded ca-certificate file path at:

https://github.com/ouspg/trytls/blob/master/stubs/lua5.1-luasec/run.lua#L47

This will fail on many operating systems. Could we e.g. have better logic for finding
system ca-bundle (hopefully built-into lua already) and fail with `UNSUPPORTED` instead
of err'ing if it is not found?

**Comment by oherrala** [comment]
Maybe the CA bundle finding logic should be in trytls runner instead of each and every stub. I think this is related to #87.

Runner could do tests like:
1. Connect to www.google.com:443, expect verify success
2. Connect to www.google.com:443 with OS's CA bundle, expect verify success

Then:
- If both, 1. and 2. fail, there's something seriously wrong.
- If only 1. fails, it's indication that stub doesn't have correct CAs. Runner could provide path to it?
- If only 2. fails, it indicates stub doesn't support CA bundles.

**Comment by ikisusi** [comment]
On the other hand people who use those languages and libraries have to have the means to use ca-bundles. They don't have an external program to do it for them.

**Comment by ikisusi** [comment]
Let me rephrase. If runner would always override CA-bundle then we would not catch all the errors there are with respect to CA-bundle handling/discovery/bundling that are native to that library and language.

**Comment by aleksiklasila** [comment]
If however a library/language does not implement native handling/discovery/bundling methods - the kind of way proposed by @oherrala sounds better to me than implementing the logic required into stubs (some of which may very well be tests for under development libraries). The bundles can be given as arguments (as the runner would) by external programs (runner or humans) if they are to use/test/develop such libraries.

**Comment by Mamietti** [comment]
Luasec does not seem to have native support. So....how do we do this?

**Comment by ikisusi** [comment]
Some options are:
- don't even try, return `UNSUPPORTED` always when CA-bundle is not given
- have some default paths, check if you can open any of them and return `UNSUPPORTED` if not, if you can then use that one

There might be others. :)
